### PR TITLE
[OLD] Remove :open command

### PR DESCRIPTION
**Pull request by ghost**
~~This is being done to fulfill #2548 (I think at least the `limits` section of `vi_diff.txt` could be kept, not sure where it could be moved though).~~

See #3569

### CODE REVIEW

File: runtime/doc/index.txt [hunk 1]
```diff
@@ -1362,7 +1362,6 @@ tag	      command	      action ~
 |:nunmap|	:nun[map]	like ":unmap" but for Normal mode
 |:nunmenu|	:nunme[nu]	remove menu for Normal mode
 |:oldfiles|	:ol[dfiles]	list files that have marks in the viminfo file
-|:open|		:o[pen]		start open mode (not implemented)
 |:omap|		:om[ap]		like ":map" but for Operator-pending mode
```
Comment (fdinoff) [inline runtime/doc/index.txt]: One these gets the `:o` as its short form since `:open` no longer has it. (I'm not sure which)

Comment (Pyrohh) [inline runtime/doc/index.txt]: :+1: Nice catch; It's `oldfiles`, so I'll update that.

**Comment by Pyrohh** [comment]
@fmoralesc how do you recommend changing the `:ol[dfiles]` highlighting to be `:o[ldfiles]` (mentioned https://github.com/neovim/neovim/pull/3054#discussion_r34961744)?
I'm not sure if the new structure of `syntax/vim.vim` changes anything in that regard.

**Comment by fmoralesc** [comment]
@Pyrohh I think it's enough to change it to `o[ldfiles]`.

**Comment by Pyrohh** [comment]
@fmoralesc okay, I did just that and squashed.

**Comment by ZyX-I** [comment]
I do not think that `:oldfiles` should get such a good abbreviation. `:only` is much more useful then various `:o*menu`, `:o*map` (because these are mainly used in scripts), `:options` (do not know who prefers this over `:help`, but this is definitely mainly needed when you choose what to have in vimrc) and `:ownsyntax` (do not know why this exists in first place). `:oldfiles` is also bad because it simply prints a list of files… which by default has up to one hundred entries making it unreadable. `:browse oldfiles` which has somewhat greater sense prints just the same list.

On the other hand `:only` can be typed as `<C-w>o`, so the alternate choice is reserving `:o` abbreviation for the future. The fact that some command will get `:o` abbreviation does *not* mean that you need to update documentation and syntax file.

**Comment by Pyrohh** [comment]
@ZyX-I Yes, I think that other commands would be better off with such a short abbreviation. However, I'm unsure if we should do that, as I thought that the list of commands in `ex_cmds.lua` was always supposed to be sorted; are you suggesting I move `:only` (for example) above `:oldfiles` so that it's matched first if `:o` is executed on the command line?

> The fact that some command will get :o abbreviation does not mean that you need to update documentation and syntax file.

Why do you say that? Is it because some commands are already outdated (you mentioned this https://github.com/neovim/neovim/pull/2288#issuecomment-87347514)?

**Comment by ZyX-I** [comment]
@Pyrohh List of commands is sorted on the *shortest abbreviation*. You can move some commands up if needed. Also when writing VimL to lua translator I have already seen a number of commands which have not the shortest abbreviations referenced in the help file (this is what I meant in that comment).

File: runtime/doc/editing.txt [hunk 2]
```diff
@@ -1129,8 +1129,8 @@ If you want to always use ":confirm", set the 'confirm' option.
 			|:mkvimrc|, |:mksession|, |:mkview|, |:split|,
 			|:vsplit|, |:tabe|, |:tabnew|, |:cfile|, |:cgetfile|,
 			|:caddfile|, |:lfile|, |:lgetfile|, |:laddfile|,
-			|:diffsplit|, |:diffpatch|, |:open|, |:pedit|,
-			|:redir|, |:source|, |:update|, |:visual|, |:vsplit|,
+			|:diffsplit|, |:diffpatch|, ||:pedit|, |:redir|,
```
Comment (Pyrohh) [inline runtime/doc/editing.txt]: :+1:

**Comment by Pyrohh** [comment]
The branch got deleted a while ago, so I'll open a new PR (with the milestone too, as it's probably low priority).

**Pull request closed without merging**